src/main/kotlin/platformapi/models/ChannelMeta.kt

```diff
@@ -0,0 +1,9 @@
+package platformapi.models
+
+import java.util.*
+
+data class ChannelMeta(
+    val id: UUID,
+    val name: String?,
+    val isManaged: Boolean,
```

Comment:
What's a managed channel? :D

Reply:
A managed channel is a channel that can only be altered by the platform API, but not by users.